app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class UserSerializer < ActiveModel::Serializer
+  attributes :id, :name, :display_name, :github_login, :github_url
```

Comment:
I'd be fine removing `github_url`. Less inclined to remove `display_name`. I think it's reasonable to include the `github_login`.